VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/joh-waiting-room/joh-waiting-room.component.eventhub.events.spec.ts

```diff
@@ -52,7 +52,7 @@ describe('JohWaitingRoomComponent eventhub events', () => {
             userMediaStreamService,
             notificationSoundsService
         );
-        adalService.userInfo.userName = globalParticipant.username;
+        adalService.userInfo.userName = 'chris.green@hearings.net';
```

Comment:
what happened to globalParicipant?

Reply:
The global participant (ParticipantResponse) lost the username, we removed it.